pkg/timeline/adaptor/controller/timeline.ts

```diff
@@ -112,6 +123,92 @@ export class TimelineController {
     return Result.ok(result);
   }
 
+  // ToDo: add filter,pagination
+  async getListTimeline(
+    listID: string,
+  ): Promise<
+    Result.Result<Error, z.infer<typeof GetListTimelineResponseSchema>>
+  > {
+    const notesRes = await this.listTimelineService.handle(listID as ListID);
+    if (Result.isErr(notesRes)) {
+      return notesRes;
+    }
+    const notes = Result.unwrap(notesRes);
+    const attachmentsMap = new Map<NoteID, Medium[]>();
+    const reactionsMap = new Map<NoteID, Reaction[]>();
+    const noteAuthorMap = new Map<AccountID, Account>();
+
+    const noteAuthor = new Set<AccountID>(notes.map((v) => v.getAuthorID()));
+    const noteAuthorRes = await this.accountModule.fetchAccounts([
+      ...noteAuthor,
+    ]);
+    if (Result.isErr(noteAuthorRes)) {
+      return noteAuthorRes;
+    }
+    Result.unwrap(noteAuthorRes).map((v) => noteAuthorMap.set(v.getID(), v));
+
+    for await (const v of notes) {
```

Comment:
```suggestion
    for (const v of notes) {
```
`notes` は `AsyncIterator` なオブジェクトではなく配列なので, `for await` にする意味は無いと思います.